lib/event_id_populator.rb

```diff
@@ -0,0 +1,7 @@
+class EventIdPopulator
+  def self.populate
+    EventLog::EVENTS.each do |event|
+      EventLog.where(event: event.description, event_id: nil).update_all(event_id: event.id)
```

Comment:
The only renamed one I found is covered here: https://github.com/alphagov/signonotron2/pull/440/files#diff-0fe12a07e85e0e2a9814daf5085e6653R18